templates/repo/activity.tmpl

```diff
@@ -2,7 +2,7 @@
 <div class="page-content repository commits">
 	{{template "repo/header" .}}
 	<div class="ui container">
-		<h2 class="ui header">{{.DateFrom}} - {{.DateUntil}}
+		<h2 class="ui header"><time data-date-format datetime="{{.DateFrom}}"></time> - <time data-date-format datetime="{{.DateUntil}}"></time>
```

Comment:
Would suggest `data-format="date"`. Shorter and I would generally avoid boolean attributes.